src/DynamicDataTable.jsx

```diff
@@ -524,7 +524,7 @@ class DynamicDataTable extends Component {
             </div>
         );
 
-        if (value === 'all') {
+        if (value === 'all' && this.props.renderMasterCheckbox) {
```

Comment:
Doesn't this just back it skip the if-statement then render the default checkbox after it?

Reply:
https://github.com/langleyfoxall/react-dynamic-data-table/blob/a0c04f69c2d946fa03e4e21badffc67389612fe1/src/DynamicDataTable.jsx#L527-L535